extensions/api/data-management/transferprocess/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/transferprocess/TransferProcessApiController.java

```diff
@@ -58,63 +73,103 @@ public TransferProcessApiController(Monitor monitor) {
                 .sortField(sortField)
                 .filter(filterExpression)
                 .sortOrder(sortOrder).build();
-        monitor.debug(format("get all TransferProcesses %s", spec));
-
-        return Collections.emptyList();
+        monitor.debug(format("Get all TransferProcesses %s", spec));
 
+        return service.query(spec).stream()
+                .map(tp -> transformerRegistry.transform(tp, TransferProcessDto.class))
+                .filter(Result::succeeded)
+                .map(Result::getContent)
+                .collect(Collectors.toList());
     }
 
     @GET
-    @Path("{id}")
+    @Path("/{id}")
     @Override
     public TransferProcessDto getTransferProcess(@PathParam("id") String id) {
-        monitor.debug(format("get TransferProcess with ID %s", id));
-
-        return null;
+        monitor.debug(format("Get TransferProcess with ID %s", id));
```

Comment:
could you get rid of all the log spam? no real value in that IMO...

Reply:
removed the log in the "query" methods, left in the "command" ones. Also in the other data management api controllers there are a lot of logs, probably we could cleanup them in another PR.